src/App.css

```diff
@@ -87,8 +87,8 @@ body {
   border-radius: 5px;
   border: 1px solid hsl(0, 0%, 86%);
   margin: 1rem;
-  font-size: 32px;
-  max-width: 30%;
+  /* font-size: 32px; */
```

Comment:
this is the main issue - the input field font size is setting the size of the input field while the parent divs are trying to set the width as well.

they are competing against each other.
you would need to add a smaller width value here if you want more control alongwith the font-size.